tutorial/numerical-entry.go

```diff
@@ -0,0 +1,127 @@
+---
+layout: page
+title: Numerical Entry
+
+redirect_from: /develop/numerical-entry.html
```

Comment:
This is a compatibility line, it should probably be from /tutorial/keypress-on-entry.html so people will get this page instead?

Reply:
Good point. I will change it back.